frame/elections/src/lib.rs

```diff
@@ -174,6 +188,17 @@ pub struct SeatHolder<AccountId, Balance> {
 	pub deposit: Balance,
 }
 
+/// The results of running the pre-election step.
+#[derive(Debug, Clone)]
+pub struct PreElectionResults<T: Config> {
```

Comment:
probably doesn't need to be pub.

rule of thumb: if pub, must have full documentation.